docs/examples/en/loaders/GLTFLoader.html

```diff
@@ -69,6 +69,8 @@ <h2>Extensions</h2>
 		<ul>
 			<li>[link:https://github.com/takahirox/three-gltf-extensions KHR_materials_variants]<sup>1</sup></li>
 			<li>[link:https://github.com/takahirox/three-gltf-extensions MSFT_texture_dds]</li>
+			<li>[link:https://github.com/needle-tools/three-animation-pointer KHR_animation_pointer]</li>
+			<li>[link:https://github.com/needle-tools/gltf-progressive NEEDLE_progressive]</li>
```

Comment:
Please add the comments here as well: https://github.com/mrdoob/three.js/blob/ebc584918a92fda2530009bd306992f2fb3dd23b/examples/jsm/loaders/GLTFLoader.js#L101-L103

The "old" docs will be replaced with a new one generated from the JSDoc.

Reply:
Seems to work just fine. Here's how it looks with `[NEEDLE_progressive](<url>)`

<img width="747" height="162" alt="image" src="https://github.com/user-attachments/assets/ad78b1c7-52fa-4937-92e0-cbfc649d5127" />

<img width="758" height="209" alt="image" src="https://github.com/user-attachments/assets/669138ab-2063-4c4d-a591-2a2f70e017af" />